### DISCUSSION TITLE
TCP Transport always use 7410 as logic port

### DISCUSSION THREAD

**Discussion by ulongcha** [eProsima/Fast-DDS#4245] [Support]
### Is there an already existing issue for this?

- [X] I have searched the existing issues

### Expected behavior

ServerParticipant caculate logic port as `PB + DG * domainId + offsetd1 + PG * participantId` automatically.

### Current behavior

When many ClientParticipant connect to ServerParticipant, they all use 7410 as metadata port.

### Steps to reproduce

Two ClientParticipant connect to one ServerParticipant

### Fast DDS version/commit

2.11.2

### Platform/Architecture

Ubuntu Focal 20.04 amd64

### Transport layer

TCPv4

**Comment by JesusPoderoso** [comment]
Hi @ulongcha thanks for using Fast DDS.
TCP transport is not part of the DDS standard, so the [well known port(s)](https://fast-dds.docs.eprosima.com/en/latest/fastdds/transport/listening_locators.html#well-known-ports) formula is not necessarily applied in TCP.
The only communication channel in TCP case is the physical port, but it has been included a mechanism in the last release (#4061) to automatically assign a port if it is set to `0`. 

As this is the designed behavior and not a bug, I am moving this ticket to the support discussion forum as per [Fast DDS CONTRIBUTING guidelines](https://github.com/eProsima/Fast-DDS/blob/master/CONTRIBUTING.md).
